snap/local/ratings_wrapper

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+
+export APP_JWT_SECRET=$(snapctl get app-jwt-secret)
```

Comment:
```suggestion
export APP_JWT_SECRET="$(snapctl get app-jwt-secret)"
```

Applies to all your scripts -- please quote your variables to avoid any unpredictable splitting